app/messaging/index.js

```diff
@@ -2,13 +2,18 @@ const { MessageReceiver } = require('ffc-messaging')
 const config = require('../config')
 const processFileMessage = require('./process-file-message')
 const { keepAlive } = require('../keep-alive')
+const { sendFailureEvent } = require('../event')
 let messageReceiver
 
 const start = async () => {
   if (config.enabled) {
     const messageAction = message => processFileMessage(message, messageReceiver)
     messageReceiver = new MessageReceiver(config.fileReceiverSubscription, messageAction)
-    await messageReceiver.subscribe()
+    const messageError = (args) => {
+      console.error(args.error)
+      sendFailureEvent(undefined, args.error)
```

Comment:
`sendFailureDate` is a promise so needs to be awaited.

Also might be best to swap the params in the signature so we don't need to pass `undefined` to it.

Reply:
Thanks, resolved